_layouts/tutorial_hands_on.html

```diff
@@ -177,7 +178,14 @@
             <a href="https://wiki.galaxyproject.org/Teach/GTN">Galaxy Training Network</a>
             and all the contributors
             {% if tutorial.contributors %}
-            ({{ tutorial.contributors|map: 'name'|join: ', '}})
+                {% capture contributors %}
+                    {% for contributor in tutorial.contributors %}
+                        , {{ contributors[contributor].name }}
+                    {% endfor %}
+                {% endcapture %}
+
+                ( {{ contributors | remove_first: ',' }} )
```

Comment:
Do we really need extra spaces before/after the parentheses?

Reply:
no we don't :) will change